templates/index.html

```diff
@@ -4,46 +4,264 @@
 <head>
     <meta charset="UTF-8">
     <title>Question bot</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Noto+Sans:wght@300;400;500;700&family=Open+Sans:wght@300;400;600&display=swap" rel="stylesheet">
     <link href="https://unpkg.com/tailwindcss@^2/dist/tailwind.min.css" rel="stylesheet">
+    <!-- TODO: move styles here => <link rel="stylesheet" href="../assets/style.css"> --> 
```

Comment:
wasn't sure how to get python local server to serve these up in the expected place, but something like this should be good